src/Components/Web/src/Web/EventHandlers.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Web
```

Comment:
It turns out the compiler already treats event handlers and bind attributes as scoped based on namespace. I wasn't aware this already worked, and I figured we'd have to add that support later.

This is important because it allows us control how a feature like `bind` behaves based on the usings. Suppose we had *blazor native* for some other UI stack, it won't offer completion for DOM based events because they won't be in scope, because you won't have a using for `.Web`.

It's important that this behaviour is controller by usings rather than assembly references, because user code or libraries end up with extra references all of the time by accident.